src/inabox/inabox.js

```diff
@@ -24,12 +28,18 @@ export class Inabox {
    * @param {!Window} win
    */
   constructor(win) {
-    /** @const {!Window} */
+    /** @private {!Window} */
     this.win_ = win;
   }
 
   init() {
-    this.win_.top./*OK*/postMessage('amp-inabox:hello world', '*');
+    this.win_.addEventListener('message', event => {
+      dev().info(TAG, event.data);
+    });
+
+    this.win_.top./*OK*/postMessage('amp-' + JSON.stringify({
```

Comment:
this file seems to be just for testing as it is right now. does it need to be committed at this stage vs. once it is actually fleshed out?